src/build/prepareElectronApp.ts

```diff
@@ -121,7 +121,7 @@ async function maybeCopyScripts(srcs: string[], dest: string): Promise<void> {
     }
 
     const postFixHash = generateRandomSuffix();
-    const destFileName = `inject-${postFixHash}.${path.extname(src)}`;
+    const destFileName = `inject-${postFixHash}${path.extname(src)}`;
```

Comment:
@ronjouch I added this when testing out this entry. I discovered that we were doubly injecting a `.` because `path.extname` already includes it.